cfme/containers/pod.py

```diff
@@ -52,9 +42,28 @@ def get_detail(self, *ident):
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
 
-    def navigate(self, detail=True):
-        if detail is True:
-            if not self._on_detail_page():
-                sel.force_navigate('containers_pod_detail', context={'pod': self})
-        else:
-            sel.force_navigate('containers_pod', context={'pod': self})
+
+@navigator.register(Pod, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+
+    def step(self):
+        nav._nav_to_fn('Compute', 'Containers', 'Pods')(None)
+
+    def resetter(self):
+        # Reset view and selection
+        tb.select("List View")
```

Comment:
Cool - this line appears in many places so it'll be easy to re-factor this later on